cmd/wallet-web/src/pages/OIDCShare.vue

```diff
@@ -309,6 +346,23 @@ export default {
         this.credentialDisplayData[currentCredentialType] || this.credentialDisplayData.fallback
       );
     },
+    generateIdToken: function () {
+      const header = JSON.stringify({
+        alg: 'none',
+      });
+      const payload = JSON.stringify({
+        iss: this.$route.query.client_id,
+        sub: this.$route.query.client_id,
+        aud: this.$route.query.client_id,
+        iat: Date.now(),
+        exp: Date.now(),
+      });
+      const signature = JSON.stringify({});
+      const encodedHeader = encode(header).slice(0, -1);
+      const encodedPayload = encode(payload).slice(0, -1);
+      const encodedSignature = encode(signature).slice(0, -1);
+      this.idToken = `${encodedHeader}.${encodedPayload}.${encodedSignature}`;
```

Comment:
On a side note- probably we should check for jwt libraries which we can use rather than us constructing ?

Reply:
I tried but didn't find, maybe we should still looking for one though